lib/puppet/interface/option.rb

```diff
@@ -33,20 +33,28 @@ def initialize(parent, *declaration, &block)
         # jeffweiss 17 april 2012
         name = optparse_to_optionname(item)
         if Puppet.settings.include? name then
-          raise ArgumentError, "#{item.inspect}: already defined in puppet"
+          raise ArgumentError, _("%{option}: already defined in puppet") % { option: item.inspect }
         end
         if dup = dups[name] then
-          raise ArgumentError, "#{item.inspect}: duplicates existing alias #{dup.inspect} in #{@parent}"
+          # TODO
+          # TODO
+          # TODO
+          # TODO
+          # TODO
+          # TODO
+          # TODO
```

Comment:
This is a lot of todos....

Reply:
Sorry that was a marker for me to know where to start looking again and I missing pushing up the delete for it.